src/mngt/ocf_mngt_cache.c

```diff
@@ -2183,11 +2183,15 @@ static void _ocf_mngt_activate_check_superblock_complete(void *priv, int error)
 		OCF_PL_FINISH_RET(context->pipeline, result);
 
 	if (cache->conf_meta->metadata_layout != cache->metadata.layout) {
+		ocf_cache_log(cache, log_err, "Failed to activate failover instance: "
```

Comment:
I think "standby instance" or "failover standby instance" is more precise than "failover instance" in this case